### DISCUSSION TITLE
Small app changes

### DISCUSSION THREAD

**Discussion by MTRatcliffe** [johnkeel-thork/TET#23] [Ideas]
After doing a small part of the welsh TET this weekend, the app is great and here is my happy critical feedback: 

-Since downloading the app and returning home it constantly pops up with urgent updates multiple times a day [android app], could we make the updates only appear when the app is loaded/running? 

-Is there a way to add colour to the tracks to mark difficulty [similar to mountain bike ones], a person in the riding group developed a clutch slipping issue on steep section, he would have been able to ride a forestry track but not hard green lanes. We tried to develop a track home using the elevation data [avoiding hills], I would imagine a way to tell how hard the track is would be beneficial to new riders or those that are on heavy bikes.  In the end we used the ACT tracks as these had the colour/hardness data mentioned above, but it would be a shame to push users of the TET app towards a paid for service.

**Comment by johnburgg** [comment]
best ever thread
